sdcm/nemesis.py

```diff
@@ -3990,6 +3992,26 @@ def format_error_for_sla_test_and_raise(error_events):
                                                   for error in error_events if error)
                                         )
 
+    def disrupt_create_index(self):
+        """
+        Create index on a random column (regular or static) of a table with the most number of partitions and wait until it gets build.
+        Then verify it can be used in a query. Finally, drop the index.
+        """
+        with self.cluster.cql_connection_patient(self.target_node, connect_timeout=300) as session:
+            ks, cf = get_table_with_biggest_partitions_count(session)
+            column = get_random_column_name(session, ks, cf)
+            try:
+                index_name = create_index(session, ks, cf, column)
+            except InvalidRequest as exc:
+                LOGGER.warning(exc)
+                raise UnsupportedNemesis(  # pylint: disable=raise-missing-from
+                    "Tried to create already existing index. See log for details")
+            try:
+                wait_for_index_to_be_built(self.target_node, ks, index_name, timeout=1200)
+                verify_query_by_index_works(session, ks, cf, column)
+            finally:
+                drop_index(session, ks, index_name)
```

Comment:
I think that before dropping the index, as I wrote in @yarongilor 's PR for adding MV, we should let the test run with the index for a while.
In the other PR I suggested 1-2% of the test duration, but didn't give much thought.
Maybe:
5m < 1% < 40m
(minimum 5m, 1% for the general case, maximum 40m for the very long cases).

Reply:
done